Backtrace/Model/BacktraceStackTrace.cs

```diff
@@ -40,9 +43,13 @@ private void Initialize()
             bool generateExceptionInformation = _exception != null;
             //initialize environment stack trace
             var stackTrace = new StackTrace(true);
-            //reverse frame order
-            var frames = stackTrace.GetFrames();
-            SetStacktraceInformation(frames, generateExceptionInformation);
+
+            if (_exception == null || _includeEnvironmentStackTrace)
+            {
+                //reverse frame order
+                var frames = stackTrace.GetFrames();
+                SetStacktraceInformation(frames, generateExceptionInformation);
```

Comment:
I'm really not sure that the "environment" stack will ever really be useful if you have an exception. This will just tell you who passed the exception to the Backtrace client. But more importantly, there's no separation between the exception stack and the environment stack. So you still are just smashing them together with no way to separate them when reviewing your reports.

Reply:
@bocciab we removed environment flag in next commit. Thanks for your advice!